apps/style/code-studio/petition.scss

```diff
@@ -0,0 +1,116 @@
+.petition_block {
+  width: 970px;
+  display: flex;
+  flex-flow: column wrap;
+  text-align: center;
+  background-color: white;
+  padding: 10px;
+}
+
+.petition_message {
+  margin: 0;
+  font-size: 21px;
+  line-height: 28px;
+  font-family: 'Gotham 5r', sans-serif;
```

Comment:
you should be able to include shared/css/font.scss using ```@import 'font'``` at the top and then use the variables in there so that theres more consistency

Reply:
Excellent, thanks!